pkg/transport/httprpc/httprpc.go

```diff
@@ -230,5 +233,7 @@ func (t *HTTPRPC) handleError(err error, codec channel.Codec, req *http.Request,
 		fmt.Fprint(w, "unknown error")
 	}
 
-	w.Write(payload)
+	if _, error :=w.Write(payload); error != nil {
```

Comment:
nit: run `go fmt` on this file.

Reply:
Fixed and I added a test to fail PRs if files aren't formatted going forward.